cli/cli/commands/enclave/stop/stop.go

```diff
@@ -4,11 +4,13 @@ import (
 	"context"
 	"fmt"
 	"github.com/kurtosis-tech/kurtosis/api/golang/engine/kurtosis_engine_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/lib/kurtosis_context"
```

Comment:
leaving a comment here for discussion to prevent auto merge

wonder whether StopEnclave inside Engine Server can do a service stop so that ts/curl/any other frontend gets this functionality for free

Reply:
As we talked in Slack the problem is that there is not connection between the Engine and the APIC server, so doesn't allow us to do it in the engine side